README.md

```diff
@@ -392,6 +392,110 @@ The default port the Pushgateway is listening to is 9091. The path looks like:
 * For example to wipe all metrics from the Pushgateway:
 
         curl -X PUT http://pushgateway.example.org:9091/api/v1/admin/wipe
+
+## Rest API
+
+Rest API provides Rest based interface for accessing metrics and build information. 
```

Comment:
Then this would read:

The query API allows accessing pushed metrics and build and runtime information.